test/controllers/api/jobs_controller_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'test_helper'
+
+class Api::JobsControllerTest < ActionController::TestCase
+
+  let(:user) { User.create!(email: 'test@prx.org', password: 'foobarpassword') }
+
+  let(:application) do
+    Doorkeeper::Application.create(
+      name: 'test',
+      owner: user,
+      redirect_uri: 'urn:ietf:wg:oauth:2.0:oob'
+    )
+  end
+
+  let(:token) do
+    token = Minitest::Mock.new
+    token.expect(:acceptable?, true, [Object])
+    token.expect(:resource_owner_id, user.id)
+    token.expect(:application, application)
+    token
+  end
+
+  let(:job) do
+    Job.create(job_type: 'audio', priority: 1, application_id: application.id)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.